src/StaticValidator.php

```diff
@@ -47,14 +47,22 @@ public static function getPluginManager()
     /**
      * @param  mixed    $value
      * @param  string   $classBaseName
-     * @param  array    $args          OPTIONAL
+     * @param  array    $optons OPTIONAL options to pass to the validator constructor;
```

Comment:
* typo s/optons/options
* Include "as first argument". "associative array of options for to inject in the first argument of the validator constructor"